src/lib/addons/slack-app.test.ts

```diff
@@ -102,7 +102,10 @@ describe('SlackAppAddon', () => {
             ],
         };
 
-        await addon.handleEvent(eventWith2Tags, { accessToken });
+        await addon.handleEvent(eventWith2Tags, {
+            accessToken,
+            defaultChannels: '',
```

Comment:
I guess default channels should be allowed to be undefined... maybe the type definition is wrong?